tests/tests_twobody/test_events.py

```diff
@@ -207,3 +207,103 @@ def test_penumbra_event_crossing():
     )
 
     assert expected_penumbra_t.isclose(epoch + penumbra_event.last_t, atol=1 * u.s)
+
+
+def test_orbit_propagation_continues_if_events_terminal_is_False():
+    r = [-6142438.668, 3492467.56, -25767.257] << u.km
+    v = [505.848, 942.781, 7435.922] << u.km / u.s
+    orbit = Orbit.from_vectors(Earth, r, v)
+
+    thresh_lat = 60 * u.deg
+    # Event occurs at ~1701.7 s.
+    latitude_cross_event = LatitudeCrossEvent(orbit, thresh_lat, terminal=False)
+    events = [latitude_cross_event]
+
+    # The last two tofs are after the detection of the event.
+    tofs = [1000, 1250, 1500, 1710, 2000] << u.s
+    rr, _ = cowell(
+        Earth.k,
+        orbit.r,
+        orbit.v,
+        tofs,
+        events=events,
+    )
+
+    # Check position vector of orbit doesn't hang.
```

Comment:
Instead of saying "orbit doesn't hang" let us say here "checking not repeated position vectors during propagation".